probs/cmd/probs/configs.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	probs "github.com/moonstream-to/api/probs/pkg"
+	engine "github.com/moonstream-to/api/probs/pkg/engine"
+)
+
+var (
+	DEFAULT_CONFIG_DIR_NAME  = ".probs"
+	DEFAULT_CONFIG_FILE_NAME = "config.json"
+)
+
+// Workers configuration
+type ServiceWorkersConfig struct {
+	Name    string                `json:"name"`
+	DbUri   string                `json:"db_uri"`
+	Workers []probs.ServiceWorker `json:"workers"`
+}
+
+func ReadConfig(configPath string) (*[]ServiceWorkersConfig, int, error) {
+	totalWorkersNum := 0
+
+	rawBytes, err := ioutil.ReadFile(configPath)
+	if err != nil {
+		return nil, totalWorkersNum, err
+	}
+	serviceWorkersConfigTemp := &[]ServiceWorkersConfig{}
+	err = json.Unmarshal(rawBytes, serviceWorkersConfigTemp)
+	if err != nil {
+		return nil, totalWorkersNum, err
+	}
+
+	var serviceWorkersConfig []ServiceWorkersConfig
+	for _, service := range *serviceWorkersConfigTemp {
+		serviceDbUri := os.Getenv(service.DbUri)
+		if serviceDbUri == "" {
+			return nil, totalWorkersNum, fmt.Errorf("unable to load database URI for service %s", service.Name)
+		}
+
+		var serviceWorkers []probs.ServiceWorker
+
+		// Link worker function
+		for w, worker := range service.Workers {
+			switch service.Name {
+			case "engine":
+				for _, engineWorker := range engine.ENGINE_SUPPORTED_WORKERS {
+					if worker.Name == engineWorker.Name {
```

Comment:
Not really an issue, but it would be a good idea to define a `map[string]Worker` in engine instead of iterating like this.